src/utils/line.ts

```diff
@@ -785,3 +776,45 @@ export const formatOptimisticProposal = (
 
   return proposedPosition;
 };
+
+// TODO: Move code from getLines to create state.categories into this function
+export const formatLineCategories = (
+  lines: { [key: string]: SecuredLine },
+  categories: { [key: string]: string[] }
+): { [key: string]: string[] } => {
+  const validLines = _.filter(lines, (line) => line.status === 'active' && line.end * 1000 > Date.now());
+
+  const highestCreditLines = _.map(
```

Comment:
Should invert the sort algo so that lines you want are put first instead of last so you dont need to call reverse()

Reply:
done!